streams/quickstart/java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <kafka.version>2.4.2-SNAPSHOT</kafka.version>
+        <kafka.version>5.4.4-ccs-SNAPSHOT</kafka.version>
```

Comment:
just curious, why was streams on 2.4.2 and now it jumped to 5.4.4?

Reply:
I've only "recently" started bumping these versions in these poms (hence the AK version) - @ijuma has approved this in the past.